stock_release_channel_delivery/models/stock_picking.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2023 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class StockPicking(models.Model):
+
+    _inherit = "stock.picking"
+
+    def _get_release_channel_possible_candidate_domain(self):
+        domain = super()._get_release_channel_possible_candidate_domain()
+        if self.carrier_id:
+            domain.extend(
+                [
+                    "|",
+                    ("carrier_ids", "=", False),
+                    ("carrier_ids", "=", self.carrier_id.ids),
```

Comment:
Don't you have a warning as "in" is better IMHO?

Reply:
Indeed, you have:

![image](https://github.com/OCA/wms/assets/19529533/d0419868-124e-4d2a-9d3f-356115cfcdda)